### ISSUE TITLE
Crash on add to cart

### ISSUE THREAD

**Issue by yasirmturk** [lunarphp/lunar#2019]
- Lunar version: 1.0.0-beta.5
- Laravel Version: 11.31.0
- PHP Version: 8.2.25
- Database Driver & Version: 10.6.20-MariaDB

### Expected Behaviour:

Product should be added to the cart

### Actual Behaviour:

Crash with ErrorException Attempt to read property "id" on null

### Steps To Reproduce:

Product created with 1 variant, 1 product option (Size) and one option (Small), 
Add to Card on Product page

**Comment by yasirmturk** [comment]
Once this crash happens, the app goes to unstable state, and now even the home page is not loading and crashes

<img width="947" alt="Screenshot 2024-11-17 at 20 54 43" src="https://github.com/user-attachments/assets/f7a34fba-101c-450a-a830-52cfab2a4930">

**Comment by yasirmturk** [comment]
Underlying reason for this crash is that, Tax Zone is not set in the admin. 

So the checkout does not work without setting tax zone while it is an optional configuration

![Screenshot 2024-11-17 at 22 18 14](https://github.com/user-attachments/assets/281b85e7-5455-4372-bb99-7cb733e6f8ef)

**Comment by gbaggaley** [comment]
Also getting this error and it was fixed by adding a tax zone.

**Comment by yasirmturk** [comment]
Yes, adding tax zone fixes it but then it should be a mandatory configuration
